src/controllers/recordController.ts

```diff
@@ -10,23 +10,36 @@ import {
 export class RecordController {
   async getRecordForMember(id: string) {
     const [data] = await pool.query(
-      `SELECT * FROM AttendanceRecord WHERE person_id = ?`,
+      `SELECT * FROM AttendanceRecord JOIN Event ON AttendanceRecord.event_id = Event.uuid WHERE person_id = ?`,
```

Comment:
Good idea combining the event object onto the record to save the api call to fetch that data